CONTRIBUTE.md

```diff
@@ -42,12 +42,15 @@ Run the commands (or follow the MkDocs documentation to locally pip install MkDo
 
     # requirements.txt
 
-    ## using pip
-    pip install -r requirements.txt
+    ## using pip and venv
+    python -m venv venv
```

Comment:
I would suggest:
`python -m venv .venv`

I truly don't know if it makes much of a difference, but it's what is recommended in the guidance.

This would also change the activation line to include the . in the filepath